### PR TITLE
Add scope validation to the personal info page in myaccount

### PR DESCRIPTION
**Pull request by nipunthilakshan**

### CODE REVIEW

File: apps/myaccount/src/components/profile/profile.tsx [hunk 1]
```diff
@@ -18,8 +18,9 @@
 
 import { updateProfileImageURL } from "@wso2is/core/api";
 import { ProfileConstants } from "@wso2is/core/constants";
-import { isFeatureEnabled, resolveUserDisplayName, resolveUserEmails,
-    getUserNameWithoutDomain } from "@wso2is/core/helpers";
+import { hasRequiredScopes } from "@wso2is/core/helpers";
+import { getUserNameWithoutDomain, isFeatureEnabled, resolveUserDisplayName,
```
Comment (brionmario) [inline apps/myaccount/src/components/profile/profile.tsx:22]:
```suggestion
import {
    getUserNameWithoutDomain,
    isFeatureEnabled,
    resolveUserDisplayName,
```

File: apps/myaccount/src/components/profile/profile.tsx [hunk 2]
```diff
@@ -18,8 +18,9 @@
...
+    resolveUserEmails } from "@wso2is/core/helpers";
```
Comment (brionmario) [inline apps/myaccount/src/components/profile/profile.tsx:23]:
```suggestion
    resolveUserEmails
} from "@wso2is/core/helpers";
```

File: apps/myaccount/src/components/profile/profile.tsx [hunk 3]
```diff
@@ -889,6 +893,18 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): J
         </>
     );
 
+    /**
+     * Check whether the profile url is readonly.
```
Comment (brionmario) [inline apps/myaccount/src/components/profile/profile.tsx:897]:
```suggestion
     * Check whether the profile url is readonly.
     *
     * @return {boolean}
```

**Review by brionmario — APPROVED** [review]

File: apps/myaccount/src/components/profile/profile.tsx (hunk 1 above)
Comment (nipunthilakshan) [reply apps/myaccount/src/components/profile/profile.tsx]: Addressed.

File: apps/myaccount/src/components/profile/profile.tsx (hunk 2 above)
Comment (nipunthilakshan) [reply apps/myaccount/src/components/profile/profile.tsx]: Addressed.

File: apps/myaccount/src/components/profile/profile.tsx (hunk 3 above)
Comment (nipunthilakshan) [reply apps/myaccount/src/components/profile/profile.tsx:901]: Addressed.

**Pull request merged**